### DISCUSSION TITLE
Redis cache cleaning

### DISCUSSION THREAD

**Discussion by Gsyltc** [goauthentik/authentik#8179] [Q&A]
Hello, 
Is there a way to clean the Redis cache ? I currently have over 400K entries in the cache. There are only 4 users at home. I find this number quite high.

Thanks

**Comment by BeryJu** [comment]
There was an issue fixed for this by https://github.com/goauthentik/authentik/commit/3e44e9d3f66eea26500cf0ed58c64f6861167404 however thats only in the current beta release. Otherwise it's probably easiest to stop the redis container and delete its volume

**Comment by Gsyltc** [comment]
Thank you for your quick reply.
I can't stop my cluster as I would like. There is a lot of service (9 databases in use). I will change database for the moment and delete the old one.

Thank you for your answer.
